packages/mds-metrics/request-handlers.ts

```diff
@@ -267,3 +267,32 @@ export async function getAll(req: MetricsApiRequest, res: GetAllResponse) {
     res.status(500).send(new ServerError(error))
   }
 }
+
+export async function getAllStubbed(req: MetricsApiRequest, res: GetAllResponse) {
+  const { query } = req
+  const bin_size = getBinSizeFromQuery(query)
+
+  const { start_time, end_time } = parseRelative(query.start || 'today', query.end || 'now')
+  const slices = getTimeBins({
+    bin_size,
+    start_time,
+    end_time
+  })
+  const provider_id = query.provider_id || null
+  const vehicle_type = query.vehicle_type || null
+  const format : string | 'json' | 'tsv' = query.format || 'json'
+
+  if (format !== 'json' && format !== 'tsv') {
+    return res.status(400).send(new BadParamsError(`Bad format query param: ${format}`))
+  }
+
+  if (provider_id !== null && !isUUID(provider_id))
+    return res.status(400).send(new BadParamsError(`provider_id ${provider_id} is not a UUID`))
+
+  // TODO test validation
+  if (vehicle_type !== null && !Object.values(VEHICLE_TYPES).includes(vehicle_type))
+    return res.status(400).send(new BadParamsError(`vehicle_type ${vehicle_type} is not a valid vehicle type`))
+
+  const tsvStub = await fs.readFileSync('./metrics-sample-v1.tsv')
```

Comment:
`readFileSync` is not asynchronous. No need for an `await`. You could also just use `fs.readFileSync(...).toString()` to convert the `Buffer` to a string.

Reply:
That all sounds good, will do